.eslintrc.cjs

```diff
@@ -12,7 +12,7 @@ module.exports = {
 	overrides: [
 		{
 			extends: ["plugin:markdown/recommended"],
-			files: ["**/*.{md}"],
+			files: ["**/*.md"],
```

Comment:
Heh, nice catch. If you can pull this out into a separate PR I'd be happy to pull it in quickly.

Reply:
Hmm, this is probably what's causing https://github.com/JoshuaKGoldberg/template-typescript-node-package/actions/runs/3943867603/jobs/6749189690:

```plaintext
/home/runner/work/template-typescript-node-package/template-typescript-node-package/README.md
  96:1  error  Parsing error: ESLint was configured to run on `<tsconfigRootDir>/README.md/2_2.ts` using `parserOptions.project`: <tsconfigRootDir>/tsconfig.eslint.json
```

Should probably split this out into a separate issue & PR. I can take a look at this too if you don't have time. ❤️

Reply:
Maybe it's convenient to leave it in this PR? Eventually it's caused by different backing glob parsers (in ESLint and Knip), but necessary for green lights in this PR.